docs/src/Sheet.doc.js

```diff
@@ -615,113 +599,501 @@ function SubheadingExample(props) {
+                        id="favoriteDog"
+                        label="No change"
+                        name="favorite"
+                        onChange={() => {}}
+                        value="dogs"
+                      />
+                      <RadioButton
+                        checked={false}
+                        id="favoriteCat"
+                        label="Automatic (recommended)"
+                        subtext="Pinterest aims to get the most clicks for your budget"
+                        name="favorite"
+                        onChange={() => {}}
+                        value="cats"
+                      />
+                      <RadioButton
+                        checked={false}
+                        id="favoritePlants"
+                        label="Custom"
+                        subtext="You control how much to bid at auctions"
+                        name="favorite"
+                        onChange={() => {}}
+                        value="plants"
+                      />
+                    </Flex>
+                  </Fieldset>,
+                summary: ['Custom'],
+                title: 'Bid',
+              }]}>
+          </Module.Expandable>
+          <Module.Expandable
+            accessibilityExpandLabel="Expand the module"
+            accessibilityCollapseLabel="Collapse the module"
+            id="ModuleExample - preview"
+            items={[
+              {
+                children: <Text> Preview table of changes here</Text>,
+                summary: ['5 ad groups changing'],
+                title: 'Preview bid changes',
+              }]}>
+          </Module.Expandable>
+        </Flex>
+      </Sheet>
+    );
+  };
+
+  const [shouldShow, setShouldShow] = React.useState(true);
+  const HEADER_ZINDEX = new FixedZIndex(10);
+  const sheetZIndex = new CompositeZIndex([HEADER_ZINDEX]);
+
+  return (
+    <React.Fragment>
+      <Button
+        inline
+        text="View footer example"
+        onClick={() => setShouldShow(true)}
+      />
+      {shouldShow && (
+        <Layer zIndex={sheetZIndex}>
+          <SheetWithFooter onDismiss={() => setShouldShow(false)} />
+        </Layer>
+      )}
+    </React.Fragment>
+  );
+}
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Sizes"
+      description={`
+Sheet comes in 3 sizes: small (\`sm\`), medium (\`md\`), and large (\`lg\`).
+- Small Sheets (540px) are primarily used for displaying information or acting as a point to link to other content. They are the least commonly used.
+- Medium Sheets (720px) are the standard size offered for content.
+- Large Sheets (900px) should be used in cases where there may be columns of content or navigation where the additional space is required to keep the content at a comfortable reading width.
+`}
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+function SizesExample(props) {
```

Comment:
Possible future improvement: use different content for each size to better exemplify when each size should be used

Reply:
Yeah I was definitely planning to update the Large size once we have SideNav :D

Reply:
Product tagging on Pinbuilder is a pretty neat `md` example that's not so form-heavy (makes good use of the sub-heading and footer as well)